tests/show_techsupport/test_techsupport.py

```diff
@@ -457,3 +483,64 @@ def test_techsupport_commands(
         )
 
     pytest_assert(len(cmd_not_found) == 0, cmd_not_found)
+
+def test_secret_removed_from_show_techsupport(
+    duthosts, enum_rand_one_per_hwsku_hostname, creds_all_duts, check_image_version, setup_password
+):
+    """
+    This test checks following secrets been removed from show techsupport result:
+        Tacacs key
+        Radius key
+        snmp community string
+        /etc/shadow, which includes the hash of local/domain users' password
+        /etc/sonic/*.certs, which are Azure internal used certificates
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+
+    tacacs_passkey = creds_all_duts[duthost]['tacacs_passkey']
+    radius_passkey = creds_all_duts[duthost]['radius_passkey']
+    snmp_rocommunity = creds_all_duts[duthost]['snmp_rocommunity']
+
+    # generate a new dump file
+    duthost.shell('sudo rm -rf /var/dump/sonic_dump_*')
```

Comment:
>rm [](http://example.com/codeflow?start=24&length=2)

Suggest not destroy dump files others created.

Reply:
Fixed